<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bryła 3D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>70</y>
      <width>427</width>
      <height>427</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>70</y>
      <width>140</width>
      <height>427</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Translacja">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Przesunięcia:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Skalowanie">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Skalowanie:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rotacja">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotacja:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="g1_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>34</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="g1">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="x1">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y1">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="z1">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSlider" name="translacjaX">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>-400</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="translacjaY">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>-400</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="translacjaZ">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>-400</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>149</y>
       <width>20</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="x2">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="z2">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSlider" name="skalowanieX">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="skalowanieY">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="skalowanieZ">
        <property name="styleSheet">
         <string notr="true">background:#CCFFCC;</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>258</y>
       <width>31</width>
       <height>135</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="x3">
        <property name="text">
         <string>OX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="y3">
        <property name="text">
         <string>OY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="z3">
        <property name="text">
         <string>OZ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="g6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>120</width>
       <height>135</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QDial" name="rotacjaX">
        <property name="styleSheet">
         <string notr="true">background: #22FF77</string>
        </property>
        <property name="maximum">
         <number>628</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="rotacjaY">
        <property name="styleSheet">
         <string notr="true">background: #22FF77</string>
        </property>
        <property name="maximum">
         <number>628</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="rotacjaZ">
        <property name="styleSheet">
         <string notr="true">background: #22FF77</string>
        </property>
        <property name="maximum">
         <number>628</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>579</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>369</x>
       <y>0</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #22FF77; border-radius: 1px;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quit">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>0</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #22FF77; border-radius: 1px;</string>
     </property>
     <property name="text">
      <string>Wyjście</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>frame</zorder>
   <zorder>groupBox1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
